src/github/graphql.ts

```diff
@@ -309,4 +311,66 @@ export interface RateLimit {
 	cost: number;
 	remaining: number;
 	resetAt: string;
+}
+
+export interface PullRequestListItem {
+	nodeId: string;
+	number: number;
+	title: string;
+	url: string;
+	author: {
+		login: string;
+		url: string;
+		avatarUrl: string;
+	};
+	state: string;
+	assignees: {
+		nodes: IAccount[];
+	};
+	reviewRequests: {
+		nodes: [{
+			requestedReviewer: IAccount;
+		}];
+	};
+	createdAt: string;
+	updatedAt: string;
+	merged: boolean;
+	headRef: {
```

Comment:
Can we somehow reuse `interface Ref`

Reply:
Or probably a new `interface RefParts`

Reply:
Yea, it's good enough to re-use `Ref` thanks.